pkg/server/server.go

```diff
@@ -36,159 +36,176 @@ import (
 	PluginProxy           string
+
 	// Clients with the correct TLS setup for communicating with the API servers.
-	LocalK8sClient                      *http.Client
-	LocalK8sProxyConfig                 *proxy.Config
-	ThanosProxyConfig                   *proxy.Config
-	ThanosTenancyProxyConfig            *proxy.Config
-	ThanosTenancyProxyForRulesConfig    *proxy.Config
 	AlertManagerProxyConfig             *proxy.Config
-	AlertManagerUserWorkloadProxyConfig *proxy.Config
 	AlertManagerTenancyProxyConfig      *proxy.Config
+	AlertManagerUserWorkloadProxyConfig *proxy.Config
+	ClusterManagementProxyConfig        *proxy.Config
+	GitOpsProxyConfig                   *proxy.Config
+	K8sClient                           *http.Client
+	ServiceClient                       *http.Client
+	K8sProxyConfig                      *proxy.Config
 	MeteringProxyConfig                 *proxy.Config
-	TerminalProxyTLSConfig              *tls.Config
 	PluginsProxyTLSConfig               *tls.Config
-	GitOpsProxyConfig                   *proxy.Config
-	ClusterManagementProxyConfig        *proxy.Config
-	ManagedClusterProxyConfig           *proxy.Config
+	TerminalProxyTLSConfig              *tls.Config
+	ThanosProxyConfig                   *proxy.Config
+	ThanosTenancyProxyConfig            *proxy.Config
+	ThanosTenancyProxyForRulesConfig    *proxy.Config
+
 	// A lister for resource listing of a particular kind
-	MonitoringDashboardConfigMapLister ResourceLister
-	KnativeEventSourceCRDLister        ResourceLister
-	KnativeChannelCRDLister            ResourceLister
 	GOARCH                             string
 	GOOS                               string
+	KnativeChannelCRDLister            ResourceLister
+	KnativeEventSourceCRDLister        ResourceLister
+	MonitoringDashboardConfigMapLister ResourceLister
+
 	// Monitoring and Logging related URLs
+	AddPage                      string
 	AlertManagerPublicURL        *url.URL
-	GrafanaPublicURL             *url.URL
-	PrometheusPublicURL          *url.URL
-	ThanosPublicURL              *url.URL
-	AlertManagerUserWorkloadHost string
 	AlertManagerTenancyHost      string
+	AlertManagerUserWorkloadHost string
+	CopiedCSVsDisabled           bool
 	DevCatalogCategories         string
 	DevCatalogTypes              string
-	UserSettingsLocation         string
-	QuickStarts                  string
-	AddPage                      string
-	ProjectAccessClusterRoles    string
+	GrafanaPublicURL             *url.URL
 	Perspectives                 string
+	ProjectAccessClusterRoles    string
+	PrometheusPublicURL          *url.URL
+	QuickStarts                  string
 	Telemetry                    serverconfig.MultiKeyValue
-	CopiedCSVsDisabled           bool
-	HubConsoleURL                *url.URL
+	ThanosPublicURL              *url.URL
+	UserSettingsLocation         string
+
+	// multicluster
+	Authers                   map[string]*auth.Authenticator
+	HubConsoleURL             *url.URL
+	ManagedClusterProxyConfig *proxy.Config
+}
```

Comment:
Everything above here was just reorganizing these vars into alphabetical blocks, and separating the public constants from private ones.